Autopilot/Path Manager/PathManager.c

```diff
@@ -340,7 +340,23 @@ char followWaypoints(PathData* currentWaypoint, float* position, float heading,
 
             *sp_Heading = (int)followStraightPath((float*)&waypointDirection, (float*)targetCoordinates, (float*)position, heading);
         }
-        else{
+        else{            
```

Comment:
@iehink My bad.  I just took another look and see that you did it correctly. Apologies, I must have been mentally handicapped at that moment.